install.vbs

```diff
@@ -70,7 +70,16 @@ End If
 ' Copy the hard files to where their symlinks would be.
 Call CopySymlinkFiles()
 
-WScript.Echo vbCrLf & "Successfully initialized submodules and fixed symlinks."

```

Comment:
That's the code for windows developer (but it hasn't been tested - I don't have a windows box).